kiota_abstractions/base_request_configuration.py

```diff
@@ -20,3 +21,5 @@ class BaseRequestConfiguration:
 
     # Request options
     options: Optional[List[RequestOption]] = None
+
+    query_parameters: Optional[GetQueryParameters] = None
```

Comment:
My suggestion here if that makes sense in python:
- Add a QueryParameters class that has everything GetQueryParameters had
- Mark GetQueryParameters as deprecated/obsolete
- Empty GetQueryParameters
- Make GetQueryParameters inherit from QueryParameters
- Update all references to GetQueryParameters to QueryParameters in the libs but also in the generation

This way older clients that pull the updated dependency will still work (but get a warning hopefully)
Newer clients get the correct naming.
We can delete GetQueryParameters in v2.


Thoughts?

Reply:
Yeah, this should work. Will be both backward compatible and make python aligned with other languages in terms of naming.